lib/drives/forms/create_drive_form.dart

```diff
@@ -75,6 +79,8 @@ class _CreateDriveFormState extends State<CreateDriveForm> {
             _selectedDate.day, _selectedDate.hour + 2, _selectedDate.minute);
         final Profile driver = SupabaseManager.getCurrentProfile()!;
 
+        print("$_startSuggestion $_destinationSuggestion");
```

Comment:
Why do we nee tihs?

Reply:
No great reason, just to debug while you test and as a reminder to later do something with the values - and to get the compiler to shut up about unused variables